convert/src/utils/s3_utils.py

```diff
@@ -42,6 +49,7 @@ def get_files(self, bucket_s3: str, files_keys: List[str]) -> None:
         """
         Downloads files from S3 storage
         """
+        bucket_s3 = convert_bucket_name_if_s3prefix(bucket_s3)
```

Comment:
I think we shouldn't change bucket name here since the function already takes it.

Reply:
agreed